renderdoc/driver/gl/gl_replay.cpp

```diff
@@ -405,7 +405,10 @@ void GLReplay::GetBufferData(ResourceId buff, uint64_t offset, uint64_t len, vec
 
   gl.glBindBuffer(eGL_COPY_READ_BUFFER, buf.resource.name);
 
-  gl.glGetBufferSubData(eGL_COPY_READ_BUFFER, (GLintptr)offset, (GLsizeiptr)len, &ret[0]);
+  void *data =
+      gl.glMapBufferRange(eGL_COPY_READ_BUFFER, (GLintptr)offset, (GLsizeiptr)len, eGL_MAP_READ_BIT);
+  memcpy(&ret[0], data, (size_t)len);
+  gl.glUnmapBuffer(eGL_COPY_READ_BUFFER);
```

Comment:
I think this should be emulated the same way as other unsupported functions with this map/memcpy/unmap inside, since there are other places that use glGetBufferSubData.